graph/ui-project-details/src/lib/project-details/project-details.tsx

```diff
@@ -160,16 +186,69 @@ export const ProjectDetails = ({
   );
 };
 
-export default ProjectDetails;
+// const ProjectsDropdown = ({
```

Comment:
Remove? This is in a follow-up?

Reply:
this is still WIP, will remove once it's finalized